packages/app/src/app/pages/Sandbox/QuickActions/QuickActions.tsx

```diff
@@ -17,32 +18,35 @@ import {
   Keybindings,
 } from './elements';
 
-class QuickActionsComponent extends React.Component {
+export class QuickActions extends React.Component {
```

Comment:
why is it still class based? It should be functional component

Reply:
I've done the changes, please check